3p/iframe-messaging-client.js

```diff
@@ -33,7 +33,7 @@ export class IframeMessagingClient {
     this.win_ = win;
     /** @private {?string} */
     this.rtvVersion_ = getMode().rtvVersion || null;
-    /** @private {!Window} */
+    /** @private {!(Window|Element)} */
```

Comment:
Done. (Where did you find that, so I don't have to bug you next time I need to type something as a checkbox or whatever?)